ProjectSettings/ProjectSettings.asset

```diff
@@ -13,7 +13,7 @@ PlayerSettings:
   useOnDemandResources: 0
   accelerometerFrequency: 60
   companyName: Baconeta Studios
-  productName: 'Setting the Stage '
+  productName: 'SettingTheStage '
```

Comment:
You left the space at the end !! XD

I have fixed it now

Reply:
omg shoot me in the head